--- vamp-plugin-sdk-v1.1b/vamp-sdk/PluginAdapter.h.gcc43	2007-10-19 08:52:33.000000000 -0400
+++ vamp-plugin-sdk-v1.1b/vamp-sdk/PluginAdapter.h	2008-01-31 11:26:58.000000000 -0500
@@ -41,8 +41,6 @@
 
 #include "Plugin.h"
 
-#include <map>
-
 namespace Vamp {
 
 /**
--- vamp-plugin-sdk-v1.1b/vamp-sdk/Plugin.h.gcc43	2007-10-19 08:52:33.000000000 -0400
+++ vamp-plugin-sdk-v1.1b/vamp-sdk/Plugin.h	2008-01-31 11:20:46.000000000 -0500
@@ -40,6 +40,8 @@
 #include "PluginBase.h"
 #include "RealTime.h"
 
+#include <cstdlib>
+#include <cstring>
 #include <string>
 #include <vector>
 #include <map>
--- vamp-plugin-sdk-v1.1b/vamp-sdk/PluginHostAdapter.h.gcc43	2007-10-19 08:52:33.000000000 -0400
+++ vamp-plugin-sdk-v1.1b/vamp-sdk/PluginHostAdapter.h	2008-01-31 11:17:59.000000000 -0500
@@ -40,8 +40,6 @@
 #include <vamp/vamp.h>
 #include <vamp-sdk/Plugin.h>
 
-#include <vector>
-
 namespace Vamp {
 
 /**
