--- vamp-plugin-sdk-v1.1b/Makefile.orig	2007-10-19 08:52:34.000000000 -0400
+++ vamp-plugin-sdk-v1.1b/Makefile	2008-01-28 14:13:24.000000000 -0500
@@ -28,7 +28,7 @@
 
 # Compile flags
 #
-CXXFLAGS	:= $(CXXFLAGS) -O2 -Wall -I. -fpic
+CXXFLAGS	:= $(CXXFLAGS) -I. -fpic
 
 # ar, ranlib
 #
@@ -57,11 +57,13 @@
 # editing for non-Linux platforms.  Of course you don't necessarily
 # have to use "make install".
 #
-INSTALL_PREFIX	 	  := /usr
+PREFIX                    := /usr
+LIBDIR                    := /lib
+INSTALL_PREFIX	 	  := $(DESTDIR)$(PREFIX)
 INSTALL_API_HEADERS	  := $(INSTALL_PREFIX)/include/vamp
 INSTALL_SDK_HEADERS	  := $(INSTALL_PREFIX)/include/vamp-sdk
 INSTALL_HOSTEXT_HEADERS	  := $(INSTALL_PREFIX)/include/vamp-sdk/hostext
-INSTALL_SDK_LIBS	  := $(INSTALL_PREFIX)/lib
+INSTALL_SDK_LIBS	  := $(DESTDIR)$(LIBDIR)
 
 INSTALL_SDK_LIBNAME	  := libvamp-sdk.so.1.1.0
 INSTALL_SDK_LINK_ABI	  := libvamp-sdk.so.1
@@ -75,7 +77,7 @@
 INSTALL_HOSTSDK_STATIC    := libvamp-hostsdk.a
 INSTALL_HOSTSDK_LA        := libvamp-hostsdk.la
 
-INSTALL_PKGCONFIG	  := $(INSTALL_PREFIX)/lib/pkgconfig
+INSTALL_PKGCONFIG	  := $(INSTALL_SDK_LIBS)/pkgconfig
 
 # Flags required to tell the compiler to create a dynamically loadable object
 #
@@ -212,14 +214,14 @@
 		mkdir -p $(INSTALL_HOSTEXT_HEADERS)
 		mkdir -p $(INSTALL_SDK_LIBS)
 		mkdir -p $(INSTALL_PKGCONFIG)
-		cp $(API_HEADERS) $(INSTALL_API_HEADERS)
-		cp $(SDK_HEADERS) $(INSTALL_SDK_HEADERS)
-		cp $(HOSTSDK_HEADERS) $(INSTALL_SDK_HEADERS)
-		cp $(HOSTEXT_HEADERS) $(INSTALL_HOSTEXT_HEADERS)
-		cp $(SDK_STATIC) $(INSTALL_SDK_LIBS)
-		cp $(HOSTSDK_STATIC) $(INSTALL_SDK_LIBS)
-		cp $(SDK_DYNAMIC) $(INSTALL_SDK_LIBS)/$(INSTALL_SDK_LIBNAME)
-		cp $(HOSTSDK_DYNAMIC) $(INSTALL_SDK_LIBS)/$(INSTALL_HOSTSDK_LIBNAME)
+		cp -p $(API_HEADERS) $(INSTALL_API_HEADERS)
+		cp -p $(SDK_HEADERS) $(INSTALL_SDK_HEADERS)
+		cp -p $(HOSTSDK_HEADERS) $(INSTALL_SDK_HEADERS)
+		cp -p $(HOSTEXT_HEADERS) $(INSTALL_HOSTEXT_HEADERS)
+		cp -p $(SDK_STATIC) $(INSTALL_SDK_LIBS)
+		cp -p $(HOSTSDK_STATIC) $(INSTALL_SDK_LIBS)
+		cp -p $(SDK_DYNAMIC) $(INSTALL_SDK_LIBS)/$(INSTALL_SDK_LIBNAME)
+		cp -p $(HOSTSDK_DYNAMIC) $(INSTALL_SDK_LIBS)/$(INSTALL_HOSTSDK_LIBNAME)
 		rm -f $(INSTALL_SDK_LIBS)/$(INSTALL_SDK_LINK_ABI)
 		ln -s $(INSTALL_SDK_LIBNAME) $(INSTALL_SDK_LIBS)/$(INSTALL_SDK_LINK_ABI)
 		rm -f $(INSTALL_SDK_LIBS)/$(INSTALL_HOSTSDK_LINK_ABI)
@@ -228,11 +230,11 @@
 		ln -s $(INSTALL_SDK_LIBNAME) $(INSTALL_SDK_LIBS)/$(INSTALL_SDK_LINK_DEV)
 		rm -f $(INSTALL_SDK_LIBS)/$(INSTALL_HOSTSDK_LINK_DEV)
 		ln -s $(INSTALL_HOSTSDK_LIBNAME) $(INSTALL_SDK_LIBS)/$(INSTALL_HOSTSDK_LINK_DEV)
-		sed "s,%PREFIX%,$(INSTALL_PREFIX)," $(APIDIR)/vamp.pc.in \
+		sed "s,%PREFIX%,$(PREFIX)," $(APIDIR)/vamp.pc.in \
 		> $(INSTALL_PKGCONFIG)/vamp.pc
-		sed "s,%PREFIX%,$(INSTALL_PREFIX)," $(SDKDIR)/vamp-sdk.pc.in \
+		sed "s,%PREFIX%,$(PREFIX)," $(SDKDIR)/vamp-sdk.pc.in \
 		> $(INSTALL_PKGCONFIG)/vamp-sdk.pc
-		sed "s,%PREFIX%,$(INSTALL_PREFIX)," $(SDKDIR)/vamp-hostsdk.pc.in \
+		sed "s,%PREFIX%,$(PREFIX)," $(SDKDIR)/vamp-hostsdk.pc.in \
 		> $(INSTALL_PKGCONFIG)/vamp-hostsdk.pc
 		sed -e "s,%LIBNAME%,$(INSTALL_SDK_LIBNAME),g" \
 		    -e "s,%LINK_ABI%,$(INSTALL_SDK_LINK_ABI),g" \
